sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultServiceSpec.java

```diff
@@ -147,6 +148,22 @@ public ReplacementFailurePolicy getReplacementFailurePolicy() {
     }
 
     @Override
+    public Boolean getGpuOptin() {
```

Comment:
Would like this moved off of the `DefaultServiceSpec` and moved to the `DefaultService` or similar.